scriptmodules/supplementary/runcommand.sh

```diff
@@ -19,7 +19,7 @@ function _update_hook_runcommand() {
 }
 
 function depends_runcommand() {
-    local depends=()
+    local depends=(python-pyudev)
```

Comment:
> the dependency may be better to put in the main retropie-setup script.

The dependencies for RetroPie-Setup are coded in which file?

> I was also thinking we could do it without the module perhaps.

Well, the package is really small:
```
The following NEW packages will be installed:
  python-pyudev
(...)
Need to get 31.6 kB of archives.
After this operation, 193 kB of additional disk space will be used.
```

I think it's doable without pyudev using `os.system()` (or other python function to execute shell commands) and executing those commands you used in [configedit.sh](https://github.com/RetroPie/RetroPie-Setup/blob/master/scriptmodules/supplementary/configedit.sh#L114). But IMHO it would be ugly and non-pythonic - and, as an aspiring python programmer, I need to be proud to do things in a pythonic way :-)